src/components/styles/Salary.styled.js

```diff
@@ -0,0 +1,13 @@
+import styled from "styled-components";
+export const StyledSalary = styled.section`
+  border-radius: 1em;
+  margin-left: 5%;
+  width: 90%;
+
+  .both-payouts,
```

Comment:
Nice choice of structure, it would be wise to implement this in Housing as you said.